dirsrvtests/tests/suites/indexes/regression_test.py

```diff
@@ -124,6 +131,38 @@ def fin():
 
     request.addfinalizer(fin)
 
+
+@pytest.fixture(scope="function")
+def set_description_index(request, topo, add_a_group_with_users):
+    """
+    Set some description values and description index without reindexing.
+    """
+    inst = topo.standalone
+    backends = Backends(inst)
+    backend = backends.get(DEFAULT_BENAME)
+    indexes = backend.get_indexes()
+    attr = 'description'
+
+    def fin(always=False):
+        if always or not DEBUGGING:
+            try:
+                idx = indexes.get(attr)
+                idx.delete()
+            except ldap.NO_SUCH_OBJECT:
+                pass
+
+    request.addfinalizer(fin)
+    fin(always=True)
+    index = indexes.create(properties={
```

Comment:
It might be worth using `ensure_state` instead of `create` here.
https://github.com/389ds/389-ds-base/blob/main/src/lib389/lib389/_mapped_object.py#L1065

Reply:
Not in this specific case:
    We have to explicitly delete the index if it exists (and is indexed) or step 5 will assert
     (because the index is already indexed while the test expect it is not the case)